crates/holochain/src/conductor/conductor/tests/agent_key_revocation.rs

```diff
@@ -834,6 +835,43 @@ fn assert_delete_agent_key_present_in_source_chain(
         .test_read(move |txn| txn.query_row(sql, [], row_fn).unwrap());
 }
 
+fn assert_delete_agent_key_accepted_by_validation(
+    agent_key: &AgentPubKey,
+    conductor: &SweetConductor,
+    dna_hash: &DnaHash,
+) {
+    let sql = "\
+        SELECT Action.author, Action.type, Action.deletes_entry_hash
+        FROM Action
+        JOIN DhtOp On DhtOp.action_hash = Action.hash
+        WHERE DhtOp.validation_status = :valid_status
+        AND Action.deletes_entry_hash IS NOT NULL
+        ORDER BY seq DESC";
+    conductor.get_dht_db(dna_hash).unwrap().test_read({
+        let agent_key = agent_key.clone();
+        move |txn| {
+            let mut stmt = txn.prepare(sql).unwrap();
+            let rows: Vec<_> = stmt
+                .query_map(
```

Comment:
You changed it from query to query_map, which does return an error if empty :+1: